lib/webpacker/compiler.rb

```diff
@@ -58,17 +58,21 @@ def record_compilation_digest
     def run_webpack
       logger.info "Compiling…"
 
-      stdout, sterr , status = Open3.capture3(
+      stdout, stderr, status = Open3.capture3(
         webpack_env,
         "#{RbConfig.ruby} ./bin/webpack",
         chdir: File.expand_path(config.root_path)
       )
 
-      if sterr == "" && status.success?
+      if status.success?
         logger.info "Compiled all packs in #{config.public_output_path}"
-        logger.info stdout if config.webpack_compile_output?
+        logger.error "#{stderr}\n" unless stderr.empty?
       else
-        logger.error "Compilation failed:\n#{sterr}\n#{stdout}"
+        logger.error "Compilation failed:\n#{stderr}"
```

Comment:
Why does the `else` case have no `\n` after `#{stderr}`?

Reply:
Hmm I think I've been used to loggers appending their own newlines. Is that not the case here?

Reply:
I only asked because there's a discrepancy. We append a new line in the `if` case but not in the `else` case. Both of the places use `logger.error()`.